app/assets/javascripts/reducers/index.js

```diff
@@ -26,7 +28,8 @@ const reducer = combineReducers({
   revisions,
   ui,
   userCourses,
-  userProfile
+  userProfile,
+  recentEdits
```

Comment:
please sort them alphabetically ;) 'recentEdits' should appear before 'revisions'